site/docs/configuration.md

```diff
@@ -156,5 +156,5 @@ df.write
 | target-file-size-bytes | As per table property      | Overrides this table's write.target-file-size-bytes          |
 | check-nullability      | true                       | Sets the nullable check on fields                            |
 | snapshot-property._custom-key_    | null            | Adds an entry with custom-key and corresponding value in the snapshot summary  |
-| partitioned.fanout.enabled       | false        | Overrides this table's write.partitioned.fanout.enabled  |
+| fanout-enabled       | false        | Overrides this table's write.spark.partitioned.fanout.enabled  |
```

Comment:
Thanks! @aokolnychyi, @RussellSpitzer, and I were just talking about making this change yesterday.

We talked about using `fanout-enabled` like you have here to match the other write options, but we came to consensus on using `write.spark.fanout.enabled` for the table property. You're probably familiar with the rationale for using `spark`. We also didn't see the need for `partitioned` in the property since it only makes sense for partitioned writes. We could think of it as a global option that only has a behavior change when the table is partitioned.

What do you think about using `write.spark.fanout.enabled` instead of `write.spark.partition.fanout.enabled`?

Reply:
@rdblue, I had a chat with @aokolnychyi earlier today and  I got to know the rationale behind adding `spark`.
I will rename it to `write.spark.fanout.enabled`